include/wx/private/smartrowheightcache.h

```diff
@@ -0,0 +1,142 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/private/smartrowheightcache.h
```

Comment:
I'm a bit uncomfortable about using "smart" in this file name. First because it looks like it's something that Julian Smart would have written and second because it's not really that smart in the first place. Let's just call it `rowheightcache`.